apitest/src/main/resources/injicertify/SunBirdCNegative/GetCredentialSunBirdCNeg/GetCredentialSunBirdCNeg.yml

```diff
@@ -0,0 +1,808 @@
+GetCredentialSunBirdC:
+   InjiCertify_GetCredentialSunBirdC_IdpAccessToken_invalid_format:
+      endPoint: $INJICERTIFYINSURANCEBASEURL$/v1/certify/issuance/credential
+      uniqueIdentifier: TC_injicertify_credentialissuance_01      
+      description: Get sunbird VC when invalid format is given      
+      role: resident
+      checkErrorsOnlyInResponse: true
+      restMethod: post
+      validityCheckRequired: true
+      inputTemplate: injicertify/SunBirdC/GetCredentialSunBirdC/GetCredentialSunBirdC
```

Comment:
input template is pointing to different folder

Reply:
Updated